scripts/opentelemetry/appsignals_custom_otel_setup/custom-opentelemetry.yaml

```diff
@@ -0,0 +1,591 @@
+            new_value: RuntimeMetric
+            aggregation_type: ""
+            experimental_scale: 0
+            label: ""
+            label_value: ""
+
+    exporters:
+      otlphttp:
+        #traces_endpoint field follows this pattern https://xray.<region>.amazonaws.com/v1/traces
+        traces_endpoint: https://xray.us-east-1.amazonaws.com/v1/traces
+        auth:
+          authenticator: sigv4auth
+
+      awsemf:
+        region: 'us-east-1'
+        log_group_name: "/aws/application-signals/custom"
+        namespace: "ApplicationSignalsCustomPoM"
+        dimension_rollup_option: "NoDimensionRollup"    
+
+      awsemf/application_signals:
+        region: 'us-east-1'
+        log_group_name: "/aws/application-signals/data"
+        namespace: "ApplicationSignals"
+        dimension_rollup_option: "NoDimensionRollup"
+        version: "1"
+        local_mode: true
+        metric_declarations:
+          - dimensions:
+              - [ Environment, Service ]
+            label_matchers:
+              - label_names:
+                  - Telemetry.Source
+                regex: '^RuntimeMetric$'
+                separator: ;
+            metric_name_selectors:
+              - '^.*$'
+
+    service:
+      pipelines:
+        traces/otel:
+          receivers: [otlp]
+          processors: [awsapplicationsignals]
+          exporters: [otlphttp]
+        metrics:
+          receivers: [otlp]
+          processors: 
+          - metricstransform/application_signals
+          - awsapplicationsignals
+          - resourcedetection
+          exporters: [awsemf/application_signals]
+        metrics/2:
+          receivers: [otlp]
+          exporters: [awsemf]
+      extensions: [sigv4auth]
+      telemetry:
+        logs:
+          level: debug
+---
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: my-instrumentation
+spec:
+  # OTEL_EXPORTER_OTLP_ENDPOINT
+  exporter:
+    endpoint: http://applicationsignals-otel-collector:4318
+  nodejs:
+    image: public.ecr.aws/n7z4r8f9/adot-js-amznlinux:latest
+    env:
+      # - name: OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+      #   value: http://appsignals-collector:4316/v1/traces
+      - name: OTEL_AWS_APPLICATION_SIGNALS_ENABLED
+        value: "false"
+      # - name: OTEL_AWS_APPLICATION_SIGNALS_EXPORTER_ENDPOINT
+      #   value: http://appsignals-collector:4316/v1/metrics
+      - name: OTEL_EXPORTER_OTLP_PROTOCOL
+        value: "http/protobuf"
+  java:
+    image: public.ecr.aws/aws-observability/adot-autoinstrumentation-java:v1.32.5
+    # image: ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-java:1.32.1
+    env:
+      # - name: OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+      #   value: http://appsignals-collector:4316/v1/traces
+      # - name: OTEL_AWS_APPLICATION_SIGNALS_ENABLED
+      #   value: "false"
+      # - name: OTEL_AWS_APPLICATION_SIGNALS_EXPORTER_ENDPOINT
+      #   value: http://appsignals-collector:4316/v1/metrics
```

Comment:
Are all these commented out lines (in this file and the other files) kept just to provide extra examples?
I suggest we provide some comments about when a user would need these configurations that are commented out.

Reply:
For consistency, the Collector URL in the comments should be `http://applicationsignals-otel-collector:4316...`